imitation/util/trainer.py

```diff
@@ -56,6 +53,33 @@ def init_trainer(env_id, use_gail=False,
                               scale=discrim_scale, **reward_kwargs)
     discrim = discrim_net.DiscrimNetAIRL(rn, **discrim_kwargs)
 
-  trainer = Trainer(env, gen_policy, discrim,
-                    expert_policies=expert_policies, **trainer_kwargs)
+  ro_paths = util.get_dump_paths(
+    env,
+    basedir=rollouts_dir,
+    policy_class=gen_policy.__class__,
+    n_dumps=n_rollout_dumps)
+  expert_rollouts = _joined_expert_rollouts(ro_paths)
+  trainer = Trainer(env, gen_policy, discrim, expert_rollouts,
+                    **trainer_kwargs)
   return trainer
+
+
+def _joined_expert_rollouts(npz_paths: Sequence[str],
```

Comment:
What use case did you have in mind for this? I can't see why we'd need it with synthetic experts: we can always just make a larger `npz` rollout dump. Maybe it'd be useful with human data, where we might want one file from each subject or each session, and then sometimes merge them.

Reply:
I wanted to directly mirror of what Justin's training script did, in hopes that this will make replicating benchmarks easier.

I think that Justin's data_collect script saves rollouts every N epochs and then the train script loading 5 rollout files as expert demonstration.

Reply:
Looking to mimic what Justin's train script did (I think this is what he did after loading 5 expert rollouts, but I'm not sure)

Reply:
It might be a good idea, given that we still want to match the data loading scenario I described above, to have `data_collect.py` save a joined NPZ file as a big dump rather than having multiple NPZs.

Reply:
@AdamGleave: Using data from different steps in training  didn't seem necessary for AIRL Cartpole, Pendulum, and Half-Cheetah. I think I'll just scrap any logic or plans for logic that joins rollouts from several experts. Maybe add this down the line if we want it later.

Reply:
I'm fine with scrapping this logic. Although since you've already written it, and it could be useful, I'm also OK with keeping this function.

Reply:
Ended up keeping `_joined_rollouts` after moving it to `util/rollout.py`, because I chose globs over single paths.